src/lib.rs

```diff
@@ -8,6 +8,12 @@
 //! for Rust.
 //!
 //! ```rust,no_run
```

Comment:
Setting this to `toml` didn't highlight the block but setting it to `rust` did, so I left it like this. Happy to change it.